korman/operators/op_nodes.py

```diff
@@ -47,6 +47,47 @@ def invoke(self, context, event):
         context.window_manager.fileselect_add(self)
         return {"RUNNING_MODAL"}
 
+pyAttribArgMap= {
+        "ptAttribute":
+            ["vislistid", "visliststates"],
```

Comment:
Should these values be sets instead of lists?

Reply:
Order is important.  This is the argument order from the function signatures.

Reply:
Makes sense